build_tools/bazel_to_cmake/bazel_to_cmake_targets.py

```diff
@@ -36,7 +36,7 @@
     # MLIR
     "@llvm-project//mlir:AllPassesAndDialects": ["MLIRAllDialects"],
     "@llvm-project//mlir:AffineToStandardTransforms": ["MLIRAffineToStandard"],
-    "@llvm-project//mlir:CFGTransforms": ["MLIRSCFToStandard"],
```

Comment:
I probably have to check into LLVM BUILD file to correct IREE BUILD and mapping rules.